src/components/landing/GroupLogin.jsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import LandingCard from './LandingCard.jsx';
+import TextInput from '../TextInput.jsx';
+import Button from '../Button.jsx';
+import '../../scss/landing.scss';
+
+/// callBack should be passed from landing page. Used to re-render the landing page
+/// with the default landing page card, which should be the login card.
+
+function GroupLogin({ callBack }) {
+  let input;
+
+  function submitCode() {
+    // TODO: Make login request
+  }
```

Comment:
```suggestion
  function submitCode(event) {
   event.preventDefault();
    // TODO: Make login request
  }
```
This ensures the page doesn't reload after a form is submitted.